docs/design/design-patterns/non-functional-requirements_investigation_guide.md

```diff
@@ -0,0 +1,148 @@
+
+While this is always a good _general_ rule to follow in software engineering, non-functional requirements for projects undertaken in highly-regulated environments (Financial Services, Gov, Healthcare) frequently have __outsized influence__ on the design and development of software systems. This makes their introduction later in the SDLC significantly more disruptive and expensive.  This increases even _further_ the importance of capturing these non-functional requirements as early in the engagement as possible when executing within a highly-regulated context.
+
+To support the process of capturing a project's _comprehensive_ non-functional requirements, this document offers a taxonomy for non-functional requirements and provides a framework for their identification, exploration, and eventual codification into formal engineering requirements as input to subsequent solution design.
+
+## Investigation Process
+
+1. Identify/brainstorm likely areas/topics requiring further investigation/definition
+1. Identify CUSTOMER stakeholder(s) responsible for each identified area/topic
+1. Schedule debrief/requirements definition session(s) with each stakeholder
+    * as necessary to achieve sufficient understanding of the probable impact of each requirement to the project
+    * both current/initial milestone and long-term/road map
+1. Document requirements/dependencies identified and related design constraints
+1. Evaluate current/near-term planned milestone(s) through the lens of the identified requirements/constraints
+    * Categorize each requirement as affecting immediate/near-term milestone(s) or as applicable instead to the longer-term road map/subsequent milestones
+1. Adapt plans for current/near-term milestone(s) to accommodate immediate/near-term-categorized requirements
+
+## Structure of Outline/Assignment of Responsible Stakeholder
+
+***
+In the following outline, assign name/email of 'responsible stakeholder' for each element after the appropriate level in the outline hierarchy.  Assume inheritance model of responsibility assignment: stakeholder at any ancestor (parent) level is also responsible for descendent (child) elements unless overridden at the descendent level).
+
+e.g.,
+
+* Parent1 *[Susan/susan@domain.com]*
+  * child1
+  * child2 *[John/john@domain.com]*
+    * grandchild1
+  * child3
+* Parent2 *[Sam/sam@domain.com]*
+  * child1
+  * child2
+
+In the preceding example, 'Susan' is responsible for `Parent1` and all of its descendants _except_ for `Parent1/child2` and `Parent1/child2/grandchild1`  (for which 'John' is the stakeholder).  'Sam' is responsible for the entirety of `Parent2` and all of its descendants.
+
+This approach permits the retention of the logical hierarchy of elements themselves while also flexibly interleaving the 'stakeholder' identifications within the hierarchy of topics if/when they may need to diverge due to e.g., CUSTOMER organizational nuances.
+
+***
+
+## Areas of Investigation
```

Comment:
You mention that this section is an awkward fit under `design-patterns`, and discoverability might not be exactly what we're looking for right away. We might do well to add links to and from other parts of the playbook to drive consumption of these non-functional requirements.

In particular, I think linking to [security](https://github.com/microsoft/code-with-engineering-playbook/tree/main/docs/security) might be a good fit, as well as [observability](https://github.com/microsoft/code-with-engineering-playbook/tree/main/docs/observability).
Adding an optional section to the [Engineering Fundamentals Checklist](https://github.com/microsoft/code-with-engineering-playbook/blob/main/docs/ENG-FUNDAMENTALS-CHECKLIST.md) could also get more eyes on these requirements.

Reply:
Links added.  Also link to doc from 'root' `readme.md` in `/docs/design/` folder.  This might be preferable instead from the `readme.md` in the `/docs/design/design-patterns/` folder.   If so, please comment accordingly and will adjust.  Else, if acceptable as-is am leaving this comment OPEN for you close accordingly.

Reply:
Closing/resolving as-is (new edits).   If other inter-document linkages were intended by comment, please open add'l/new comment thread accordingly  --  or of course just re-open this one :)